### DISCUSSION TITLE
Pending email, cant found the error...

### DISCUSSION THREAD

**Discussion by leoborlot** [postalserver/postal#2610] [Help with using Postal]
hi people. 

I´ve deploying a new postal server. when I try to send email for a delivery test, this was in pedding mode. 

<img width="576" alt="image" src="https://github.com/postalserver/postal/assets/20361821/7d476bb4-81a2-4c9f-83b3-e90c93df6c2a">

Whan I check the worker log, look a no error, but the email don send...

<img width="837" alt="image" src="https://github.com/postalserver/postal/assets/20361821/cc581b88-fcf1-485e-8e25-5acd0f32d0e3">

----------

There a any kind of 'log levels' to see whit workers dont process the messages?

**Comment by leoborlot** [comment]
I´ve remove database and reinstall postal, ans this works.

**Comment by willpower232** [comment]
It would normally be worth confirming that all parts of Postal are functioning with `postal status`, it is possible that the requeuer struggled to start and that might explain why no messages were sending.

Unfortunately RabbitMQ takes a little while to get going after boot so it might be a random error you will get in the future.
